### PR TITLE
feat: add outputs claimable banner in activity list

### LINKED ISSUE #820: ️🛠 - Add `Outputs claimable` banner in activity list
### Task description

_No response_

### Requirements

_No response_

### PR DESCRIPTION
**Pull request by MarkNerdi**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/wallet/tab-section/activity/ActivityTab.svelte [hunk 1]
```diff
@@ -38,6 +42,16 @@
             >
         </div>
     </header-row>
+    {#if amountClaimableTransactions}
+        <info-section class="flex flex-row items-center">
+            <Icon name={IconName.Bell} size="sm" customColor="yellow-600" />
```
Comment (Tuditi) [inline packages/desktop/views/dashboard/wallet/tab-section/activity/ActivityTab.svelte]: Not the right icon:

image

File: packages/desktop/views/dashboard/wallet/tab-section/activity/ActivityTab.svelte [hunk 2]
```diff
@@ -67,5 +81,10 @@
             @apply grid;
             grid-template-columns: 2fr 1fr 1fr 1fr;
         }
+
+        info-section {
+            @apply px-5 py-1.5 gap-2;
+            background-color: rgba(212, 161, 0, 0.1);
```
Comment (Tuditi) [inline packages/desktop/views/dashboard/wallet/tab-section/activity/ActivityTab.svelte]: You can now use the opacity of the collor like this:  
```suggestion
            @apply bg-yellow/10;
```

File: packages/shared/src/lib/core/activity/actions/getClaimableActivities.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        const isTimelocked = activity.asyncData?.asyncStatus === ActivityAsyncStatus.Timelocked
+
+        const isActivityIncomingAndUnclaimed =
+            activity.asyncData &&
+            (activity.direction === ActivityDirection.Incoming ||
+                activity.direction === ActivityDirection.SelfTransaction) &&
+            activity.asyncData?.asyncStatus === ActivityAsyncStatus.Unclaimed
+
+        return !isTimelocked && isActivityIncomingAndUnclaimed
+    })
```
Comment (Tuditi) [inline packages/shared/src/lib/core/activity/actions/getClaimableActivities.ts:17]: Are you going to use this function for something else than getting the length?

File: packages/shared/src/locales/en.json [hunk 4]
```diff
@@ -351,6 +351,7 @@
                 "action": "Action",
                 "address": "Address",
                 "amount": "Amount",
+                "claimableTransactions": "You have {amount} {amount, select, 1 {transaction} other {transactions}} that requires claiming",
```
Comment (Tuditi) [inline packages/shared/src/locales/en.json]:
```suggestion
                "claimableTransactions": "You have {amount} {amount, select, 1 {transaction that requires} other {transactions that require}}  claiming",
```

**Review by Tuditi — DISMISSED** [review]
Requested some changes. Nice work though :muscle:

File: packages/desktop/views/dashboard/wallet/tab-section/activity/ActivityTab.svelte (hunk 2 above)
Comment (MarkNerdi) [reply packages/desktop/views/dashboard/wallet/tab-section/activity/ActivityTab.svelte]: ah yes, that wasnt available yet at that point. ty

File: packages/shared/src/lib/core/activity/actions/getClaimableActivities.ts (hunk 3 above)
Comment (MarkNerdi) [reply packages/shared/src/lib/core/activity/actions/getClaimableActivities.ts:17]: yes, when we implement the button  "Claim All" we need the activities themself

File: packages/desktop/views/dashboard/wallet/tab-section/activity/ActivityTab.svelte (hunk 2 above)
Comment (nicole-obrien) [reply packages/desktop/views/dashboard/wallet/tab-section/activity/ActivityTab.svelte]: also warning instead of yellow 🙏

**Review by nicole-obrien — APPROVED** [review]
With the blocker on bloom-ui releases we can add the correct icon later, this looks good enough for now

**Pull request merged**